sentry/src/main/java/io/sentry/util/DebugMetaPropertiesApplier.java

```diff
@@ -11,33 +12,34 @@ public final class DebugMetaPropertiesApplier {
   public static @NotNull String DEBUG_META_PROPERTIES_FILENAME = "sentry-debug-meta.properties";
 
   public static void applyToOptions(
-      final @NotNull SentryOptions options, final @Nullable Properties debugMetaProperties) {
+      final @NotNull SentryOptions options, final @Nullable List<Properties> debugMetaProperties) {
     if (debugMetaProperties != null) {
-      applyProguardUuid(options, debugMetaProperties);
       applyBundleIds(options, debugMetaProperties);
+      applyProguardUuid(options, debugMetaProperties);
     }
   }
 
   private static void applyBundleIds(
-      final @NotNull SentryOptions options, final @NotNull Properties debugMetaProperties) {
+      final @NotNull SentryOptions options, final @NotNull List<Properties> debugMetaProperties) {
     if (options.getBundleIds().isEmpty()) {
-      final @Nullable String bundleIdStrings =
-          debugMetaProperties.getProperty("io.sentry.bundle-ids");
-      options.getLogger().log(SentryLevel.DEBUG, "Bundle IDs found: %s", bundleIdStrings);
-      if (bundleIdStrings != null) {
-        final @NotNull String[] bundleIds = bundleIdStrings.split(",", -1);
-        for (final String bundleId : bundleIds) {
-          options.addBundleId(bundleId);
+      for (Properties properties : debugMetaProperties) {
+        final @Nullable String bundleIdStrings = properties.getProperty("io.sentry.bundle-ids");
+        options.getLogger().log(SentryLevel.DEBUG, "Bundle IDs found: %s", bundleIdStrings);
+        if (bundleIdStrings != null) {
+          final @NotNull String[] bundleIds = bundleIdStrings.split(",", -1);
+          for (final String bundleId : bundleIds) {
+            options.addBundleId(bundleId);
+          }
         }
       }
     }
   }
 
   private static void applyProguardUuid(
-      final @NotNull SentryOptions options, final @NotNull Properties debugMetaProperties) {
-    if (options.getProguardUuid() == null) {
+      final @NotNull SentryOptions options, final @NotNull List<Properties> debugMetaProperties) {
+    if (options.getProguardUuid() == null && debugMetaProperties.size() > 0) {
       final @Nullable String proguardUuid =
-          debugMetaProperties.getProperty("io.sentry.ProguardUuids");
+          debugMetaProperties.get(0).getProperty("io.sentry.ProguardUuids");
```

Comment:
`m` I presume this is fine as there should never be more than one `debug-meta.properties` on Android but I guess it'd be safer to iterate over the list until we find a proguard ID. Would still be first one wins but I guess that's OK, can always improve later.